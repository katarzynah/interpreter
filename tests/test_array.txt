program f;

var x: array [3,2,5] of int;

procedure z(y: array [3] of int;);
begin
    print(x);
end;

begin
print(x[0,1,1]);
x[0,1,1,] := 13;
x[0,1,0,] := 15;
x[0,1,2,] := 3;
print(x[0,1,1,]);
print(x[0,1,]);
z(x,);

end.