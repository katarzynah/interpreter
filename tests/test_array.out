undefined
13
fromList [15,13,3,undefined,undefined]
fromList [fromList [fromList [undefined,undefined,undefined,undefined,undefined],fromList [15,13,3,undefined,undefined]],fromList [fromList [undefined,undefined,undefined,undefined,undefined],fromList [undefined,undefined,undefined,undefined,undefined]],fromList [fromList [undefined,undefined,undefined,undefined,undefined],fromList [undefined,undefined,undefined,undefined,undefined]]]
